README.md

```diff
@@ -0,0 +1,28 @@
+[![Build Status](https://travis-ci.org/RIOT-OS/applications.svg?branch=master)](https://travis-ci.org/RIOT-OS/applications)
+
+# RIOT Applications
+
+This repository provides more applications for the [RIOT operating system][riot-repo].
+Some of them are just useful tools for development work, others showcase
+software provided with RIOT beyond the simple examples within the RIOT codebase.
+
+## Usage
+
+To build and use them follow [the instructions in the RIOT repository][getting-started]
+and the respective application's README.
+
+The submodule to the RIOT repository in this repository will always point to the
+latest release. To change the RIOT version to build against (e.g. current
+master), clone the RIOT repository in a separate repository and point the
+`RIOTBASE` environment variable there:
+
+```sh
+# assuming you are in the working directory of your local clone of this repo
+cd ..
+git clone git@github.com:RIOT-OS/RIOT.git
```

Comment:
mhm, wouldn't it be easier to simply go into the submodule path and do `git fetch --all && git checkout <commit>` so no need for an extra clone and (re)setting `RIOTBASE`?

Reply:
yeah sure, maybe extend the description to show both options. Like `Either set/update the submodule to the desired commit/release or ...` separate clone.

However, add a note that changing the RIOTBASE or specifically building against another commit than the latest release might not work or show errors, as we do not test against these